pandas-stubs/core/series.pyi

```diff
@@ -232,25 +235,27 @@ class Series(IndexOpsMixin[S1], NDFrame):
         index: Axes | None = ...,
         *,
         dtype: TimestampDtypeArg,
-        name: Hashable | None = ...,
+        name: Hashable = ...,
         copy: bool = ...,
     ) -> TimestampSeries: ...
```

Comment:
Mypy says that the first two overloads overlap with incompatible return types even though they return exactly the same type! (The other overlapping errors might be legit - I have not yet ignored them)